lib/server.js

```diff
@@ -1309,13 +1309,19 @@ Server.prototype._routeErrorResponse = function _routeErrorResponse(
             return;
         }
 
-        // Expose only knonw errors
-        if (_.isNumber(err.statusCode)) {
+        // only automatically send errors that are known (e.g., restify-errors)
+        if (err && _.isNumber(err.statusCode)) {
```

Comment:
This will evaluate to true if the user does `throw true;`

Not sure we are wanting to protect against folks throwing `Boolean`s.